src/Adapter/MSTest.TestAdapter/Execution/TestMethodRunner.cs

```diff
@@ -288,13 +288,38 @@ private bool ExecuteDataSourceBasedTests(List<TestResult> results)
         else
         {
             var testDataSources = _testMethodInfo.GetAttributes<Attribute>(false)?.OfType<UTF.ITestDataSource>();
+            Stopwatch watch = new();
```

Comment:
I think this shall be removed.

Reply:
![image](https://github.com/microsoft/testfx/assets/24208132/01940e2d-2bcd-43c7-a115-d860759b208e)
I think it's needed as we calc time
and it's the same in the data row above

Reply:
You can always put null zero as the test is skipped. @nohwnd am I wrong?